src/engraving/libmscore/figuredbass.cpp

```diff
@@ -1230,7 +1230,8 @@ void FiguredBass::layoutLines()
         // probably be implemented.
     }
 
-    int i, len;
+    SystemIdx i;
```

Comment:
Please add default values to these variables

Reply:
There are a lot of places where there is no default value. I fixed it somewhere. But this is beyond the scope of the current task, so I did not edit everywhere.